com.unity.netcode.gameobjects/Tests/Editor/Serialization/FastBufferReaderTests.cs

```diff
@@ -0,0 +1,2192 @@
+using System;
```

Comment:
Between this file and the equivalent Writer file there's >4000 lines, and only 150 differences... Does a base class or shared utility class make sense here to get a bit more DRY?

Reply:
Yeah... I actually originally started by making FastBufferReaderTests be a child of FastBufferWriterTests but it wasn't really working out and I didn't want to spin my wheels on it. For some reason it didn't occur to me that making them both inherit from a shared base would work. D'oh.